packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -60,8 +62,12 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
     },
   };
 
-  const { promise: filesToDeleteAfterUpload, resolve: resolveFilesToDeleteAfterUpload } =
-    createFilesToDeleteAfterUploadPromise();
+  let _resolveFilesToDeleteAfterUpload:
```

Comment:
I wonder if we should create a utility that does this, but then does a node version check to run [`Promise.withResolvers`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/withResolvers)